### ISSUE TITLE
Enable PV for registry storage backend.

### ISSUE THREAD

**Issue by praveenkumar** [code-ready/snc#115]
By default, installer use `EmptyDir` for registry storage and this is not suitable during stop=>start scenario where user have some images to imagestream. We already have 30 pv, one of them can be used for registry side.

```
$ oc get deployment image-registry -oyaml | grep -A3 -i volumes
  volumes:
  - emptyDir: {}
    name: registry-storage
```

**Comment by praveenkumar** [comment]
Fixed by #116

**Comment by praveenkumar** [comment]
Looks like we need to change the `configs.imageregistry.operator.openshift.io` as mentioned in the https://docs.openshift.com/container-platform/4.2/registry/configuring-registry-storage/configuring-registry-storage-baremetal.html documents.

**Comment by praveenkumar** [comment]
Fixed by #121
